rules/remote-desktop-manager/rule.md

```diff
@@ -32,6 +32,6 @@ To install the application, check https://devolutions.net/remote-desktop-manager
 
 ### Reference:
 
-SugarLearning item - https://my.sugarlearning.com/SSW/items/13222/ssw-remote-desktop-manager?term=RDM
+Checkout the [SugarLearning item](https://my.sugarlearning.com/SSW/items/13222/ssw-remote-desktop-manager?term=RDM)
```

Comment:
I think this should be removed. As if I try to view this item I get a 403 error.

Reply:
I agree it should be removed - This is a public rule and should be made generic enough for the public to get some
value out of it

Reply:
Done - @bradystroud mate you made a good point. Thank you @JackDevAU 
I have removed the Reference section